components/charts/utils/styling.ts

```diff
@@ -36,7 +36,9 @@ export const getTargetLineStyling = (options: Partial<LineChartDataset> = {}): P
 export const getTrendLineStyling = (options: Partial<LineChartDataset> = {}): Partial<LineChartDataset> => ({
   borderColor: TREND_BORDER_COLOR,
   backgroundColor: TREND_BG_COLOR,
+  tension: 0,
   borderDash: [5, 5],
-  tension: 0.3,
+  pointRadius: 0,
+  borderWidth: 2,
```

Comment:
These changes were from the current trend line styling implementation shown in components/charts/ElectricityCapacityChart.tsx.. Adding them to this reusable func for consistency